io.catenax.shared.recycling_strategy_certificate/2.0.0/RecyclingStrategyCertificate.ttl

```diff
@@ -0,0 +1,108 @@
+   samm:properties ( :recyclingStrategyCertificateProperty ) ;
+   samm:operations ( ) ;
+   samm:events ( ) .
+
+:recyclingStrategyCertificateProperty a samm:Property ;
+   samm:preferredName "Recycling strategy certificate property"@en ;
+   samm:characteristic :RecyclingStrategyCertificateCharacteristic .
+
+:RecyclingStrategyCertificateCharacteristic a samm:Characteristic ;
+   samm:preferredName "Recycling strategy certificate characteristic"@en ;
+   samm:dataType :RecyclingStrategyCertificateEntity .
+
+:RecyclingStrategyCertificateEntity a samm:Entity ;
+   samm:preferredName "Recycling Strategy Certificate"@en ;
+   samm:properties ( :issuer :issueDate :catenaXIdPreviousLife :catenaXId :articleNumber :vin :oeNumber ) .
+
+:issuer a samm:Property ;
+   samm:preferredName "Issuer"@en ;
+   samm:description "BPMN of the business partner who creates this certificate"@en ;
+   samm:characteristic :IssuerCharacteristic ;
+   samm:exampleValue "BPNLBMW000000001" .
+
+:issueDate a samm:Property ;
+   samm:preferredName "Issue date"@en ;
+   samm:description "This is the date when the certificate was issued."@en ;
+   samm:characteristic samm-c:Timestamp ;
+   samm:exampleValue "2022-02-03T14:48:54.709Z"^^xsd:dateTime .
+
+:catenaXIdPreviousLife a samm:Property ;
+   samm:preferredName "Catena X ID previous life"@en ;
+   samm:description "Twin identification previous life"@en ;
+   samm:characteristic :CatenaXIdTraitCharacteristic ;
+   samm:exampleValue "urn:uuid:055c1128-0375-47c8-98de-7cf802c3241d" .
+
+:catenaXId a samm:Property ;
+   samm:preferredName "Catena-X Identifier"@en ;
+   samm:description "The Catena-X ID of the given part (e.g. the assembly), valid for the Catena-X dataspace."@en ;
+   samm:characteristic :CatenaXIdTraitCharacteristic ;
+   samm:exampleValue "urn:uuid:055c1128-0375-47c8-98de-7cf802c3241d" .
+
+:articleNumber a samm:Property ;
+   samm:preferredName "Article Number"@en ;
+   samm:description "Unique identification assigned by the authorized company for internal referencing"@en ;
+   samm:characteristic samm-c:Text ;
+   samm:exampleValue "1003467581006" .
+
+:vin a samm:Property ;
+   samm:preferredName "VIN"@en ;
+   samm:description "Vehicle Identification Number. Remark: this should be deducted from a reference to the vehicle TWIN"@en ;
+   samm:characteristic samm-c:Text ;
+   samm:exampleValue "WBA41DU060S228332" .
+
+:oeNumber a samm:Property ;
+   samm:preferredName "OE number"@en ;
+   samm:description "original equipment number"@en ;
+   samm:characteristic samm-c:Text ;
+   samm:exampleValue "736 49 28 - 02" .
+
+:IssuerCharacteristic a samm:Characteristic ;
+   samm:preferredName "Issuer characteristic"@en ;
+   samm:dataType xsd:string .
+
+:CatenaXIdTraitCharacteristic a samm-c:Trait ;
+   samm:description "Trait to ensure UUID v4 data format"@en ;
+   samm-c:baseCharacteristic :UUIDCharacteristic ;
```

Comment:
Here also you can make use of existing characteristic and don't need to define a new one.
For reference: https://github.com/eclipse-tractusx/sldt-semantic-models/pull/405/files#diff-8294717bc9be9d276f548ae9f83e2831503589ef17d4d5bf6f1463cdb397d00eR45

Reply:
This comes from the previous original version. I am also not sure if can be easily replaced. Do you think the two  feature teams (certificate team and Pi10 new model team) need a meeting for this naming issue?